### PR TITLE
Refactoring the main build script

### PR DESCRIPTION
**Pull request by YuriSorochkin**

### CODE REVIEW

File: device/ayya/main.sh [hunk 1]
```diff
@@ -1,13 +1,32 @@
...
 ROOT_DIR=MTK_R0MP1
 KERNEL_DIR=kernel-4.14
 LK_DIR=vendor/mediatek/proprietary/bootable/bootloader/lk
 
+# Repository used in rosalinux
+KERNEL_REPO=git@github.com:rosalinux/linux-4.14.189_mtk_android.git
+LK_REPO=git@github.com:rosalinux/ayya-lk.git
+
+# Repository brunch used in rosalinux
+BRUNCH=openmandriva
```
Comment (alexandertrofimov) [inline device/ayya/main.sh:13]: brunch -> branch
BRUNCH -> BRANCH

File: device/ayya/main.sh [hunk 2]
```diff
@@ -28,31 +47,144 @@ function check_directory_exist() {
...
+		usage
+		exit
+		;;
+	esac
+
+	check_directory_exist ${SOURCE_DIR}
+
+	echo "${BRUNCH} ${REPO} ${SOURCE_DIR}"
+
+	git clone -b ${BRUNCH} ${REPO} ${SOURCE_DIR}
```
Comment (alexandertrofimov) [inline device/ayya/main.sh:79]: BRUNCH -> BRANCH

File: device/ayya/main.sh [hunk 3]
```diff
@@ -28,31 +47,144 @@ function check_directory_exist() {
...
+}
+
+build() {
+	case $1 in
+	kernel)
+		echo "Build kernel"
+		if ! [ -d $ROOT_DIR/$KERNEL_DIR ]; then
+			echo "Kernal source code is not cloned"
```
Comment (alexandertrofimov) [inline device/ayya/main.sh:87]: Kernal -> Kernel

File: device/ayya/main.sh (hunk 1 above)
Comment (YuriSorochkin) [reply device/ayya/main.sh:13]: fixed

File: device/ayya/main.sh (hunk 2 above)
Comment (YuriSorochkin) [reply device/ayya/main.sh:79]: fixed

File: device/ayya/main.sh (hunk 3 above)
Comment (YuriSorochkin) [reply device/ayya/main.sh:87]: fixed

**Comment by YuriSorochkin** [comment]
Fixed on other pull request

**Pull request closed without merging**